.bazelrc

```diff
@@ -3,8 +3,8 @@
 # This lets us glob() up all the files inside the examples to make them inputs to tests
 # (Note, we cannot use `common --deleted_packages` because the bazel version command doesn't support it)
 # To update these lines, run tools/update_deleted_packages.sh
-build --deleted_packages=examples/custom_test_runner,examples/custom_test_runner/Sources/CustomTestRunner,examples/custom_test_runner/Sources/CustomTestRunnerLib,examples/custom_test_runner/Tests/CustomTestRunnerTests,examples/custom_test_runner/integration_tests,examples/custom_test_runner/integration_tests/workspace,examples/env_var_with_rootpath/sample_workspace,examples/simple,examples/simple/mockascript,examples/simple/mockascript/private,examples/simple/tests,tests/e2e_tests/workspace,tests/e2e_tests/workspace/child_workspace,tests/params_tests/workspace
-query --deleted_packages=examples/custom_test_runner,examples/custom_test_runner/Sources/CustomTestRunner,examples/custom_test_runner/Sources/CustomTestRunnerLib,examples/custom_test_runner/Tests/CustomTestRunnerTests,examples/custom_test_runner/integration_tests,examples/custom_test_runner/integration_tests/workspace,examples/env_var_with_rootpath/sample_workspace,examples/simple,examples/simple/mockascript,examples/simple/mockascript/private,examples/simple/tests,tests/e2e_tests/workspace,tests/e2e_tests/workspace/child_workspace,tests/params_tests/workspace
+build --deleted_packages=examples/custom_test_runner,examples/custom_test_runner/integration_tests,examples/custom_test_runner/integration_tests/workspace,examples/custom_test_runner/Sources/CustomTestRunner,examples/custom_test_runner/Sources/CustomTestRunnerLib,examples/custom_test_runner/Tests/CustomTestRunnerTests,examples/env_var_with_rootpath/sample_workspace,examples/simple,examples/simple/mockascript,examples/simple/mockascript/private,examples/simple/tests,tests/e2e_tests/workspace,tests/e2e_tests/workspace/child_workspace,tests/params_tests/workspace
```

Comment:
Why did this change? Something from the tidy script?

Reply:
Yeah, this was from tidy.

Reply:
There is a task in `//:tidy` that ensures that list is up-to-date. Perhaps, it was out-of-sync.

Reply:
On second look, that does not look right. Maybe something is wrong with the script. 👀